samples/sheets/insertColumn

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const {google} = require('googleapis');
+const sampleClient = require('../sampleclient');
+
+const sheets = google.sheets({
+  version: 'v4',
+  auth: sampleClient.oAuth2Client,
+});
+
+async function runSample(spreadsheetId, sheetId, startIndex, endIndex) {
+  const res = await sheets.spreadsheets.batchUpdate({
+    spreadsheetId,
+    resource: {
+      requests: [
+        {
+          insertDimension: {
+            range: {
+              sheetId,
+              dimension: "COLUMNS",
+              startIndex,
+              endIndex
+            },
+            inheritFromBefore: false,
+          }
+        }
+      ]
+    }
+  });
+}
+
+const scopes = [
+  'https://www.googleapis.com/auth/spreadsheets'
+];
+
+if (module === require.main) {
+  const [spreadsheetId, sheetId, startIndex, endIndex] = process.argv.slice(2);
```

Comment:
It would be good to tell someone the expected arguments if they do not provide the appropriate # of arguments:

I like the approach taken here:

https://github.com/googleapis/google-api-nodejs-client/blob/master/samples/drive/download.js#L62